RELEASE.md

```diff
@@ -0,0 +1,359 @@
+the tag message should be like <q>Libplanet 1.2.3</q>.
+
+~~~~ bash
+git tag -m "Libplanet 1.2.3" 1.2.3
+~~~~
+
+If you have a GPG (PGP) key, it is recommended to sign the tag using your key:
+
+~~~~ bash
+git tag --sign -m "Libplanet 1.2.3" 1.2.3
+~~~~
+
+Note that signed tags are visually distinct on GitHub:
+
+![Signed tags appearing distinct on GitHub.](https://git.io/JTRJs)
+
+The created tag should be directly pushed to the upstream repository.
+A tag push triggers GitHub Actions to release a new version.  The script
+automates the below processes:
+
+ -  Listing a new version on [GitHub Releases].  The corresponding changelog
+    section is extracted to fill up the description of the release.
+    Each release contains:
+     -  NuGet packages (*Libplanet\*.nupkg*)
+     -  Complete assemblies (*Libplanet\*-net\*.tar.xz*) for .NET Core and
+        .NET Framework/Mono
+     -  Prebuilt binaries of Libplanet Tools for Linux
+        (*planet-\*-linux-\*.tar.xz*), macOS (*planet-\*-osx-\*.tar.xz*),
+        and Windows (*planet-\*-win-\*.tar.xz*)
+ -  Submitting NuGet packages of a new version.  The packages are listed
+    in the [Contributing guide](CONTRIBUTING.md#projects).
+ -  Building a new version of docs and uploading it on
+    <https://docs.libplanet.net/>, which is hosted by GitHub Pages.
+    See also the *gh-pages* branch.
+ -  Submitting Libplanet.Tools to npm [@planetarium/cli].
+
+In other words, you don't have to do them by yourself.  However, you should
+check twice if all these packages and docs are successfully submitted.
+If something went wrong, start the debugging by reading the build logs on
+GitHub Actions.
+
+[@planetarium/cli]: https://www.npmjs.com/package/@planetarium/cli
+[GitHub Releases]: https://github.com/planetarium/libplanet/releases
+
+
+Preparing next
+--------------
+
+After a new tag is merged and all automated processes are successful,
+you need to prepare the next release.  This process is slightly tricky,
+because the process is vary depending on whether the released version
+was a major/minor release, or just a patch release.
+
+
+### Patch release
+
+If what you have just released is a patch release, there are two manual tasks
+to do:
+
+ -  Port the released code to the *main* branch, so that the next major/minor
+    release does not miss bugfixes from the previous patch release.
+ -  Prepare the next patch release.
+
+First of all, your last release should be ported to the *main* branch.
+Switch to the *main* branch and make sure it up-to-date.
+Then, merge the tag (e.g., *1.2.3*) you made above into the *main* branch:
+
+~~~ bash
+git switch main  # Or on Git < 2.23: git checkout main
+git fetch upstream && git reset --hard upstream/main
+git merge 1.2.3
+~~~
+
+This trying to merge will most likely have conflicts on the *CHANGES.md* file,
+because both branches (*main* and *<var>X</var>.<var>Y</var>-maintenance*) has
+their own topmost section in the changelog.  However, this can be simply
```

Comment:
```suggestion
Trying to merge like above will most likely have conflicts on the *CHANGES.md* file,
because both branches (*main* and *<var>X</var>.<var>Y</var>-maintenance*) have
their own topmost section in the changelog.  However, this can be simply
```